benchmark/fixtures/require-builtins.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const list = [
+  'async_hooks',
+  'assert',
+  'buffer',
+  'child_process',
+  'console',
+  'constants',
+  'crypto',
+  'cluster',
+  'dgram',
+  'dns',
+  'domain',
+  'events',
+  'fs',
+  'http',
+  'http2',
+  'https',
+  'inspector',
+  'module',
+  'net',
+  'os',
+  'path',
+  'perf_hooks',
+  'process',
+  'punycode',
+  'querystring',
+  'readline',
+  'repl',
+  'stream',
+  'string_decoder',
+  'timers',
+  'tls',
+  'trace_events',
+  'tty',
+  'url',
+  'util',
+  'vm',
+  'zlib',
+];
```

Comment:
Hardcode was likely missed some modules, just like `v8`, `worker_threads` here.

Can we directly use `require('module').builtinModules` ?

Reply:
@ZYSzys That way we may not be get fair results on different versions of Node.js (some of them may have more modules)